src/gwt/panmirror/src/editor/src/nodes/image/image-events.ts

```diff
@@ -71,6 +71,19 @@ function imagePaste(ui: EditorUI) {
         event.preventDefault();
         return true;
       }
+
+      // Electron specific clipboard check
+      for (const item of clipboardEvent.clipboardData.items) {
```

Comment:
Is there a specific reason for doing it this way instead of just adding another 
`else if (clipboardEvent.clipboardData.types.includes('image/png')` ?